src/shared/search/DocSearch.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {FC, memo, useMemo} from 'react'
+
+import HitComponent from './HitComponent'
+
+import {GLOBALSEARCH_API_KEY, GLOBALSEARCH_APP_ID} from 'src/shared/constants'
+
+import {DocSearchModal} from '@docsearch/react'
+import '@docsearch/react/style'
+
+import './GlobalSearch.scss'
```

Comment:
should this file be renamed to `DocSearch.scss`?

Reply:
Nope, bc there is a distinction between the global search and the widget im about to merge in

Reply:
right but it seems like the `GlobalSearch.scss` styles are imported at the child `DocSearch` component rather than the parent `GlobalSearch` component, right? These same styles will apply to `DocSearch` if it lives on its own or is being used by the GlobalSearch component.

Reply:
ah gotcha, moved it up a file